packages/client/package.json

```diff
@@ -134,7 +134,6 @@
     "jest-styled-components": "^7.1.1",
     "jsdom": "^21.1.1",
     "openfin-cli": "^4.0.0",
-    "playwright": "1.37.1",
```

Comment:
No need to `Playwright` anymore

Reply:
Something has changed since we discussed this and checked `package-lock.json` diffs locally ..
Now the `playwright` dependency is installed regardless as a _transitive_ dependency of `@playwright/test`
Check the diff - it is tiny compared to the last submission of this PR.
(which was when you replaced the `package-lock.json` rather than updating it)
Might need a closer look ..

Reply:
ok, I think this is fine, now you have the minimal updates to lock file based on removing the explicit `playwright` dependency